ggshield/core/scan/commit.py

```diff
@@ -73,6 +75,57 @@ def parser(commit: "Commit") -> Iterable[Scannable]:
 
         return Commit(sha, parser, info)
 
+    @staticmethod
+    def from_merge(
+        exclusion_regexes: Optional[Set[Pattern[str]]] = None,
+        merge_branch: str = "MERGE_HEAD",
+        cwd: Optional[Path] = None,
+    ) -> "Commit":
+
+        def get_file_sha_from_ls_tree(
+            ref: str,
+            list_diff_files: List[str] = [],
+        ) -> Iterable[Tuple[str, str]]:
+            ls_tree = git(["ls-tree", "-z", ref] + list_diff_files, cwd=cwd)
```

Comment:
naming: maybe suffix this one with `_output` to become `ls_tree_output` like you did below for the diff output?